react/src/components/AssetDetail/AssetGeometry.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import _ from 'lodash';
+import * as turf from '@turf/turf';
+import { Feature, FeatureType, getFeatureType } from '@hazmapper/types';
+
+interface AssetGeometryProps {
+  selectedFeature: Feature;
+}
+
+const AssetGeometry: React.FC<AssetGeometryProps> = ({ selectedFeature }) => {
+  if (!selectedFeature?.geometry) return null;
+
+  const bbox =
+    selectedFeature.geometry.type !== 'Point'
+      ? turf.bbox(selectedFeature)
+      : null;
+
+  const geometryType: FeatureType = getFeatureType(selectedFeature);
```

Comment:
this is the featureType. 🤔 

do you want to keep variable name and then say:

`const geometryType = selectedFeature.geometry`